src/FsAutoComplete.Core/SignatureFormatter.fs

```diff
@@ -245,7 +246,11 @@ module SignatureFormatter =
             | _ -> false
 
         let formatParameter (p:FSharpParameter) =
-            try formatFSharpType displayContext p.Type
+            try
```

Comment:
This is the primary change.  We could eventually do tuples here as well, but I found that in the TAST a parameter like `f` in the expression `let myfun (f: (int * int * int)) = ()` is represented as a `CurriedParameterArgs` list of length 1, where that single list has a length of 3. Each FSharpParameter in the list is unnamed, so we can't easily get the name of the `f` parameter here to do our usual padding and formatting....